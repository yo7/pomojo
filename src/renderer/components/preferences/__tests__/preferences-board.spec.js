import Vue from 'vue'
import Vuex from 'vuex'
import {createRenderer} from 'vue-server-renderer'
import {shallow} from 'vue-test-utils'
import PreferencesBoard from '../preferences-board.vue'

Vue.use(Vuex)

describe('PreferencesBoard', () => {
  const state = {
    preferences: {
      goal: 8
    }
  }
  const store = new Vuex.Store({state})
  const wrapper = shallow(PreferencesBoard, {store})

  it('renders correctly', () => {
    const renderer = createRenderer()
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) {
        throw new Error()
      }
      expect(str).toMatchSnapshot()
    })
  })
})
