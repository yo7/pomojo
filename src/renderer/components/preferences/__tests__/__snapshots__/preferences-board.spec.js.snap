// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreferencesBoard renders correctly 1`] = `
<div class="preferences-board">
    <!---->
    <!---->
    <!---->
</div>
`;
